docs/en/tutorials/semi_det.md

```diff
@@ -0,0 +1,285 @@
+# Tutorial 14: Semi-supervised Object Detection
+
+Semi-supervised object detection utilizes both labeled and unlabeled data for training. On the one hand, the dependence of the model on the number of detection boxes can be reduced, and on the other hand, a large amount of unlabeled data can be used to further improve the model.
+The process of semi-supervised object detection is as below:
+
+- \[Prepare and split dataset\](#Prepare and split dataset)
```

Comment:
It should show as 

- [Prepare and split dataset]()

If we cannot do that, we can omit the contents in ()

Reply:
Format conversion by `pre-commit`, `.pre-commit-config.yaml` seems to need to update the configuration of `mdformat`

Reply:
https://github.com/executablebooks/mdformat/issues/311